src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Controllers/Api/BAU/BauUsersControllerPermissionsTests.cs

```diff
@@ -1,5 +1,4 @@
-using GovUk.Education.ExploreEducationStatistics.Admin.Controllers.Api.BAU;
-using GovUk.Education.ExploreEducationStatistics.Admin.Controllers.Api.BAU.UserManagement;
+using GovUk.Education.ExploreEducationStatistics.Admin.Controllers.Api.UserManagement;
 using GovUk.Education.ExploreEducationStatistics.Admin.Security;
 using Xunit;
 using static GovUk.Education.ExploreEducationStatistics.Admin.Tests.Services.PermissionTestUtil;
```

Comment:
I think this class may need renaming now from `BauUsersControllerPermissionsTests` to something else.